.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,14 @@
-\<your comments for this PR go here\>
+Why:
+
+\<Please describe in ~1 paragraph what the value of this PR is.\>
```

Comment:
What about adding a phrase like "for the ease of your reviewers" or "for documentation" here - I think if there's a clear reason to include the info, it's easier to remember to do so!

Reply:
done, and I added a "what" section as well, to disambiguate the "what" from the "how"